.github/workflows/deploy.yml

```
@@ -70,6 +74,10 @@ jobs:
           SNOWFLAKE_ACCOUNT: ${{secrets.SNOWFLAKE_ACCOUNT}}
           SNOWFLAKE_DB_PASSWORD: ${{secrets.SNOWFLAKE_DB_PASSWORD}}
           SNOWFLAKE_DB_USER: ${{secrets.SNOWFLAKE_DB_USER}}
+          FIDESLOG__STORAGE_BUCKET_NAME: ${{secrets.FIDESLOG__STORAGE_BUCKET_NAME}}
+          FIDESLOG__STORAGE_REGION_NAME: ${{secrets.FIDESLOG__STORAGE_REGION_NAME}}
+          FIDESLOG__STORAGE_AWS_ACCESS_KEY_ID: ${{secrets.FIDESLOG__STORAGE_AWS_ACCESS_KEY_ID}}
+          AWS_SECRET_ACCESS_KEY: ${{secrets.FIDESLOG__STORAGE_AWS_SECRET_ACCESS_KEY}}
```

I need to remember if I did this for a reason, but it feels like this should also have a prefix
```suggestion
          FIDESLOG__STORAGE_AWS_SECRET_ACCESS_KEY: ${{secrets.FIDESLOG__STORAGE_AWS_SECRET_ACCESS_KEY}}
```

---

Updated to match in [335b562](https://github.com/ethyca/fideslog/pull/99/commits/335b56211defcdc02cc2312008cda4aa3a5b533a)